src/ProxyClient/Invalidation/TagsInterface.php

```diff
@@ -29,13 +29,11 @@
     public function invalidateTags(array $tags);
 
     /**
-     * Get the value for the HTTP tag header.
-     *
-     * This concatenates all tags and ensures correct encoding.
+     * Get escaped tags
      *
      * @param array $tags
      *
-     * @return string
+     * @return array
      */
     public function getTagsHeaderValue(array $tags);
```

Comment:
When using PSR-7, I think it has become the responsibility of the HTTP messaging library chosen to format the header correctly. We may want to rename this method to `getEscapedTags()`.